documents/src/pages/elements/chart.md

```diff
@@ -96,6 +96,8 @@ ef-chart {
 ```
 ::
 
+!> EFv6 uses Chart.js v2.x. Upgrade to EFv7 for Chart.js v4.x.
```

Comment:
Note that with this change, we need to deploy https://github.com/Refinitiv/refinitiv-ui/pull/857 first.
```suggestion
!> EFv6 uses Chart.js v2. [Upgrade to EFv7](http://ui.refinitiv.com/start/upgrade-guide) for Chart.js v4.
```